COMMUNITY_MEMBERSHIP.md

```diff
@@ -90,6 +100,71 @@ After the departure date, the Member(or Maintainer/Approver) and their GitHub ID
 
 Those processes ensure proper handover and recognition while providing sufficient notice to the community.
 
+## CNCF Requests and Communications
+
+As a CNCF project, Dragonfly maintains an active relationship with the Foundation.
+
+### CNCF-Related Decisions
+
+CNCF-related requests that impact the project require community involvement, including:
+
+- Project status changes (incubation, graduation)
+- Resource requests and TOC interactions
+- CNCF program participation
+- Official statements and responses
+
+### Process for CNCF Requests
+
+#### Initiating CNCF Communications
+
+1. **Proposal**: Any Maintainer creates an issue with label `cncf-request` including description, rationale, and timeline
+2. **Discussion**: 7-day minimum community discussion period
+3. **Review**: Maintainers discuss in bi-weekly meeting
+
+#### Decision Making and Approval
+
+We make the decision based on the impact of the request:
+
+- **Standard communications**: Simple majority (>50%) of active Maintainers
+- **Significant commitments**: Supermajority (≥65%) of active Maintainers
+- 7-day minimum voting period
+
+## Governance and Project Goal Changes
+
+Significant changes to governance structure or project direction require special consideration to ensure community alignment.
+
+### Types of Governance Changes
+
+This process applies to:
+
+- **Governance Document Updates**: Major revisions to governance, membership, or ladder documents
+- **Decision-Making Changes**: Modifications to voting thresholds or consensus mechanisms
+- **Role Structure Changes**: Adding/modifying community roles or responsibilities
+- **Project Goal Changes**: Updates to mission, vision, or strategic direction
+- **Repository Structure Changes**: Significant reorganization of project repositories
+
+### Governance Change Process
+
+#### Proposal and Discussion Phase
+
+1. **Proposal**: Any Maintainer creates an issue with label `governance-change` including:
+   - Description of proposed changes
+   - Rationale and expected benefits
+   - Impact analysis and migration plan
+2. **Discussion**: 14-day minimum discussion period for community input
+3. **Refinement**: Major changes restart the discussion period
+
+#### Decision and Implementation
+
+The Proposal will be discussed in bi-weekly community meeting, and the community can provide feedback and suggestions.
+
+We make the decision based on the impact of the request:
+
+- **Major Changes**: Supermajority (≥65%) of active Maintainers, 14-day voting
```

Comment:
It seems this part duplicate with above `#### Decision Making and Approval` part, merge them into one place.